src/init.cpp

```diff
@@ -956,6 +957,7 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
 
     fIsBareMultisigStd = GetBoolArg("-permitbaremultisig", true);
     nMaxDatacarrierBytes = GetArg("-datacarriersize", nMaxDatacarrierBytes);
+    nRbfPolicy = GetArg("-replacebyfee", nRbfPolicy, 1);
```

Comment:
With a hundred options out there, I can't think of any reason a second-level default was needed now, for this particular option, except to try to channel more nodes in that direction.